terra-batch-libchart/values.yaml

```diff
@@ -2,7 +2,7 @@ exports:
   common:
     cromwell:
       name: coa-cromwell-svc
-      image: broadinstitute/cromwell:85-46a7918
+      image: broadinstitute/cromwell:85-e844915
```

Comment:
Going forward this should be auto-updated, see https://github.com/broadinstitute/cromwell/pull/7014